docker/centos8/Dockerfile

```diff
@@ -355,6 +355,17 @@ RUN mkdir -p /opt/foundationdb/old && \
 
 RUN curl -Ls https://github.com/manticoresoftware/manticoresearch/raw/master/misc/junit/ctest2junit.xsl -o /opt/ctest2junit.xsl
 
+# Download Rust binaries
+ENV RUSTUP_VERSION=1.24.3 \
+    RUST_VERSION=1.59.0 \
+    RUST_ARCH=x86_64-unknown-linux-gnu
+
+RUN wget "https://static.rust-lang.org/rustup/archive/$RUSTUP_VERSION/$RUST_ARCH/rustup-init"; \
+    chmod +x rustup-init; \
+    ./rustup-init -y --no-modify-path --profile minimal --default-toolchain $RUST_VERSION --default-host $RUST_ARCH; \
+    rm rustup-init; \
+    source $HOME/.cargo/env;
```

Comment:
does this need to be sourced in the user shell? If so, it should move to the `.bashrc` section in the `devel` image.